InfluxData.Net.InfluxDb/ResponseParsers/UserResponseParser.cs

```diff
@@ -12,7 +12,7 @@ public virtual IEnumerable<User> GetUsers(IEnumerable<Serie> series)
         {
             var users = new List<User>();
 
-            if (series == null || series.Count() == 0)
+            if (series == null || series.Count() == 0 || series.Single().Values == null)
```

Comment:
Sorry, I know that it has been some time since you submitted this PR, but are you sure `.Single()` is the best option here (and not for example `.SingleOrDefault()` or even `.First()` or `.FirstOrDefault()`? Is `series` always expected to only have a single item in this case? I'm asking because `.Single()` will throw if more than one element exists. If you think it's fine, I'll merge this tomorrow.